### PR TITLE
Sector Teams Details API

### PR DESCRIPTION
**Pull request by sekharpanja**
Checking all corner cases related to each data point within Sector Team Details API.

### CODE REVIEW

**Comment by sekharpanja** [comment]
@adammalinowski done with the code tidying up, please check.

File: mi/tests/test_sector_team_details.py [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+        win1 = WinFactory(user=self.user, hvc=FuzzyChoice(TEAM_1_HVCS), date=datetime.datetime(2016, 5, 1),
+                          sector=FuzzyChoice(TEAM_1_SECTORS))
+        notification1 = NotificationFactory(win=win1)
+        notification1.created = datetime.datetime(2016, 5, 2)
+        notification1.save()
+        response1 = CustomerResponseFactory(win=win1, agree_with_win=True)
+        response1.created = datetime.datetime(2016, 5, 3)
+        response1.save()
+
+        win2 = WinFactory(user=self.user, hvc=FuzzyChoice(TEAM_1_HVCS), date=datetime.datetime(2016, 5, 1),
```
Comment (adammalinowski) [inline mi/tests/test_sector_team_details.py]: Don't worry about it for this task, but please think about how to nicely abstract creating Wins, with or without confirmations (etc) - it is a lot of hard-to-follow lines which seems like could be expressed more simply.

File: mi/tests/test_sector_views.py [hunk 2]
```diff
@@ -691,10 +347,10 @@ def test_sector_team_campaigns_1_wins_for_all_hvcs_unconfirmed(self):
             })
 
         self.expected_response["campaigns"] = sorted(campaigns, key=lambda c: (
-                                    c['totals']['progress']['confirmed_percent'],
-                                    c['totals']['progress']['unconfirmed_percent'],
-                                    c['totals']['target']),
-                                  reverse=True)
+            c['totals']['progress']['confirmed_percent'],
```
Comment (adammalinowski) [inline mi/tests/test_sector_views.py]: isn't there a function for this?

File: mi/views/sector_views.py [hunk 3]
```diff
@@ -68,21 +74,26 @@ def _get_avg_confirm_time(self, team):
...
+            'win__id',
+            'created'
+        )
+
+        confirm_times = []
+        for win_id, dates_grouper in groupby(notifications, itemgetter('win__id')):
+            earliest_dates = next(iter(dates_grouper))
+            confirm_times.append((earliest_dates['win__confirmation__created'] - earliest_dates['created']).days)
+
+        return average(confirm_times)
```
Comment (adammalinowski) [inline mi/views/sector_views.py]: Much simpler now :)
